### PR TITLE
Created new dimension and 2 measures for Account Verification SLA

### PR DESCRIPTION
**Pull request by arthurgetaround**
@kristin-bagnall-getaround For your review

### CODE REVIEW

File: ticket_metrics.view.lkml [hunk 1]
```diff
@@ -306,6 +306,15 @@ view: ticket_metrics {
     sql: ${TABLE}.reply_time_in_minutes__calendar / 60 ;;
   }
 
+  dimension: av_reply_time_in_minutes__calendar {
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
  dimension: coalesce_reply_time_and_first_resolution_time_in_minutes__calendar {
```

File: ticket_metrics.view.lkml [hunk 2]
```diff
@@ -306,6 +306,15 @@ view: ticket_metrics {
...
+    label: "AV_Reply_Time_In_Minutes__Calendar"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
```

File: ticket_metrics.view.lkml [hunk 3]
```diff
@@ -306,6 +306,15 @@ view: ticket_metrics {
...
+    description: "The number of minutes between ticket creation and the first reply from an agent for account verification tickets"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    description: "The number of minutes between ticket creation and the first reply from an agent, if it exists.  If not, the number of minutes between ticket creation and first resolved at time.  This is primarily used for the Safety Team, where responding to a customer is not always necessary."
```

File: ticket_metrics.view.lkml [hunk 4]
```diff
@@ -444,6 +453,14 @@ view: ticket_metrics {
     drill_fields: [default*]
   }
 
+  measure: count_within_2hr_sla {
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
  measure: count_coalesce_reply_time_and_first_resolution_time_calendar_meet_2_hour_SLA {
```

File: ticket_metrics.view.lkml [hunk 5]
```diff
@@ -444,6 +453,14 @@ view: ticket_metrics {
...
+    label: "Count Within 2hr SLA"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    label: "Count Coalesce Reply Time and First Resolution Time Calendar Meet 2 hour SLA"
```

File: ticket_metrics.view.lkml [hunk 6]
```diff
@@ -444,6 +453,14 @@ view: ticket_metrics {
...
+    description: "Count AV tickets replied to within 2 hours"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    description: "Count of tickets that are responded to or first resolved within the 2 hour SLA. This is primarily used for the Safety Team"
```

File: ticket_metrics.view.lkml [hunk 7]
```diff
@@ -444,6 +453,14 @@ view: ticket_metrics {
...
+    type: sum
+    sql: CASE WHEN ${av_reply_time_in_minutes__calendar} <= 120 THEN 1 ELSE NULL END ;;
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    sql: CASE WHEN ${coalesce_reply_time_and_first_resolution_time_in_minutes__calendar} <= 120 THEN 1 ELSE NULL END ;;
```

File: ticket_metrics.view.lkml [hunk 8]
```diff
@@ -624,6 +641,14 @@ view: ticket_metrics {
     sql: ${TABLE}.full_resolution_time_in_minutes__calendar/60 ;;
   }
 
+  measure: p95_av_reply_time_hours {
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
  measure: p95_coalesce_reply_time_and_first_resolution_time_in_calendar_hours {
```

File: ticket_metrics.view.lkml [hunk 9]
```diff
@@ -624,6 +641,14 @@ view: ticket_metrics {
...
+    description: "95th percentile of the number of hours it takes for an Account Verification ticket to be first replied to"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    description: "95th percentile of the number of hours it takes for a ticket to be responded to or first resolved"
```

File: ticket_metrics.view.lkml [hunk 10]
```diff
@@ -624,6 +641,14 @@ view: ticket_metrics {
...
+    type: percentile
+    percentile: 95
+    value_format: "0.#"
+    sql: ${av_reply_time_in_minutes__calendar}/60 ;;
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml]:
```suggestion
    sql: ${coalesce_reply_time_and_first_resolution_time_in_minutes__calendar}/60 ;;
```

**Review by kristin-bagnall-getaround — COMMENTED** [review]
Added some suggestions around naming and descriptions 👍

**Comment by arthurgetaround** [comment]
@kristin-bagnall-getaround committed your comments.

File: ticket_metrics.view.lkml [hunk 11]
```diff
@@ -624,6 +640,14 @@ view: ticket_metrics {
     sql: ${TABLE}.full_resolution_time_in_minutes__calendar/60 ;;
   }
 
+  measure: p95_coalesce_reply_time_and_first_resolution_time_in_calendar_hours {
+    description: "95th percentile of the number of hours it takes for a ticket to be responded to or first resolved"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml:645]:
```suggestion
    description: "95th percentile of the number of hours it takes for a ticket to be responded to or first resolved"
    group_label: "First Reply Time"
```

File: ticket_metrics.view.lkml [hunk 12]
```diff
@@ -444,6 +452,14 @@ view: ticket_metrics {
     drill_fields: [default*]
   }
 
+  measure: count_coalesce_reply_time_and_first_resolution_time_calendar_meet_2_hour_SLA {
+    label: "Count Coalesce Reply Time and First Resolution Time Calendar Meet 2 hour SLA"
+    description: "Count of tickets that are responded to or first resolved within the 2 hour SLA. This is primarily used for the Safety Team"
```
Comment (kristin-bagnall-getaround) [inline ticket_metrics.view.lkml:457]:
```suggestion
    description: "Count of tickets that are responded to or first resolved within the 2 hour SLA. This is primarily used for the Safety Team"
    group_label: "First Reply Time"
```

**Review by kristin-bagnall-getaround — APPROVED** [review]
2 additional suggestions! Otherwise, LGTM

**Comment by arthurgetaround** [comment]
Hey @xiaolong .  For your review.  This should be quick.  1 Dimension and 2 measures

File: ticket_metrics.view.lkml [hunk 13]
```diff
@@ -306,6 +306,14 @@ view: ticket_metrics {
     sql: ${TABLE}.reply_time_in_minutes__calendar / 60 ;;
   }
 
+  dimension: coalesce_reply_time_and_first_resolution_time_in_minutes__calendar {
```
Comment (horatiol) [inline ticket_metrics.view.lkml]: Not the end of the world, but really easier if it read: `reply_time_coalesced_with_first_resolution_time__in_minutes__calendar`

File: ticket_metrics.view.lkml [hunk 14]
```diff
@@ -624,6 +641,15 @@ view: ticket_metrics {
     sql: ${TABLE}.full_resolution_time_in_minutes__calendar/60 ;;
   }
 
+  measure: p95_coalesce_reply_time_and_first_resolution_time_in_calendar_hours {
+    description: "95th percentile of the number of hours it takes for a ticket to be responded to or first resolved"
+    group_label: "First Reply Time"
+    type: percentile
+    percentile: 95
+    value_format: "0.#"
```
Comment (horatiol) [inline ticket_metrics.view.lkml]: Your source dimension is using 2 decimal, and you shouldn't suddenly cut it to 1 decimal in measure. "0.##"

Comment (horatiol) [reply ticket_metrics.view.lkml]: This applies to the other measures as well.

**Review by horatiol — APPROVED** [review]
@kristin-bagnall-getaround  Can you make the requested changes before merging?

File: ticket_metrics.view.lkml (hunk 13 above)
Comment (arthurgetaround) [reply ticket_metrics.view.lkml]: changed

File: ticket_metrics.view.lkml (hunk 14 above)
Comment (arthurgetaround) [reply ticket_metrics.view.lkml]: changed

**Comment by arthurgetaround** [comment]
Hey @horatiol .  I've addressed your comments.  Please merge when you get a chance.

**Review by kroo — APPROVED** [review]

**Review by xiaolong — APPROVED** [review]

**Pull request merged**